index.html

```diff
@@ -44,129 +50,172 @@ <h1>I'm Abigael <br> Glad to see <br> you</h1>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="bullet">Back End Dev</p>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="bullet">2015</p>
+                </div>
+                <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="skills">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button type="submit">See Project</button>
             </div>
-            <p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            <ul class="skills">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
-            <button type="submit">See Project</button>
         </section>
 
-        <section class="card work">
-            <img src="images/availability-section.svg" alt="" aria-hidden="true">
-            <h2>Multi-Post <br> Stories</h2>
-            <div class="experience-section">
-                <p class="canopy">Canopy</p>
-                <img src="images/Counter.svg" alt="" aria-hidden="true">
-                <p class="bullet">Back End Dev</p>
-                <img src="images/Counter.svg" alt="" aria-hidden="true">
-                <p class="bullet">2015</p>
+        <!-- Card 2 -->
+        <section class="card card-2 work">
+
+            <img class= "mobile-img" src="images/availability-section.svg" alt="" aria-hidden="true">
+            <img class= "desktop-img" src="images/desktop-snap2.png" alt="" aria-hidden="true">
+
+            <div class="content">
+                <h2>Multi-Post Stories</h2>
+                <div class="experience-section">
+                    <p class="mobile-only canopy">Canopy</p>
+                    <p class="desktop-only canopy">Facebook</p>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="mobile-only bullet">Back End Dev</p>
+                    <p class="desktop-only bullet">Full Stack Dev Dev</p>
+                    <img src="images/Counter.svg" alt="" aria-hidden="true">
+                    <p class="bullet">2015</p>
+                </div>
+                <p class="mobile-only card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <p class="card-text desktop-only">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                <ul class="skills">
+                    <li>html</li>
+                    <li class="desktop-only">Ruby on rails</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_103](https://user-images.githubusercontent.com/69638013/166303686-35078c01-ba03-4120-94d5-947f1d13f22f.png) | ![Screenshot_105](https://user-images.githubusercontent.com/69638013/166303645-85060415-5369-44a7-bf52-7efa0d8b4d2f.png)

- [ ] Kindly fix the content of  the  li  tag with Ruby on Rails,  adjust the text to be on one line it should look like the Figma design, this only occurs in the 2nd and 4th cards  👍🏿

Reply:
@prolajumokeoni  I have opted to remove the ruby on rails in tablet view. It will only appear in desktop view. That is the only way I have seen that it fits